src/pages/sponsors/index.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import './sponsors.css'
+import { graphql } from 'gatsby'
+import Layout from '../../components/Layout'
+import PageHeader from '../../components/PageHeader'
```

Comment:
what if `PageHeader` lived inside `Layout` and was included if `title` was passed in:

```typeof title === 'string' && <PageHeader>{title}</PageHeader>```